CefSharp.BrowserSubprocess.Core/Wrapper/Browser.cpp

```diff
@@ -0,0 +1,221 @@
+// Copyright © 2015 The CefSharp Authors. All rights reserved.
```

Comment:
Should probably be 2019

Reply:
Thanks 👍 

Fixed in https://github.com/cefsharp/CefSharp/commit/3731dfa9311c78375581daf435aef4ea5e9eb14b